amplipi/streams.py

```diff
@@ -1488,7 +1486,7 @@ def info(self) -> models.SourceInfo:
                                img_url=img,
                                supported_cmds=self.supported_cmds,
                                type=self.stream_type)
-    src_config_folder = f"{utils.get_folder('config')}/srcs/{self.src}"
+    src_config_folder = f"{utils.get_folder('config')}/srcs/v{self.src}"
```

Comment:
I think this config should be moved to the corresponding virtual source directory.